projects/igniteui-angular/src/lib/core/i18n/calendar-resources.ts

```diff
@@ -0,0 +1,35 @@
+export interface ICalendarResourceStrings {
+ igx_calendar_previous_month: string;
+ igx_calendar_next_month: string;
+ igx_calendar_select_month: string;
+ igx_calendar_select_year: string;
+ igx_calendar_range_start: string;
+ igx_calendar_range_end: string;
+ igx_selected_month_is: string;
+ first_picker: string;
+ date_pickers: string;
+ of: string;
+ starts_from: string;
+ multi_selection: string;
+ range_selection: string;
+ calendar: string;
+ with: string;
+}
+
+export const CalendarResourceStringsEN: ICalendarResourceStrings = {
+ igx_calendar_previous_month: 'Previous Month',
+ igx_calendar_next_month: 'Next Month',
+ igx_calendar_select_month: 'Select Month',
+ igx_calendar_select_year: 'Select Year',
+ igx_calendar_range_start: 'Range start',
+ igx_calendar_range_end: 'Range end',
+ igx_selected_month_is: 'Selected month is',
```

Comment:
igx_selected_month? :) New component? It's either component prefixed (hint: they all should be) or not.